Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -70,15 +73,123 @@ lemma traceForm_apply_lie_apply (x y z : L) :
   · simp only [traceForm_apply_apply, LieHom.map_lie, Ring.lie_def, mul_sub, map_sub,
       ← LinearMap.mul_eq_comp]
 
+@[simp] lemma traceForm_eq_zero_of_isNilpotent [IsReduced R] [IsNilpotent R L M] :
+    traceForm R L M = 0 := by
+  ext x y
+  simp only [traceForm_apply_apply, LinearMap.zero_apply, ← isNilpotent_iff_eq_zero]
+  apply LinearMap.isNilpotent_trace_of_isNilpotent
+  exact isNilpotent_toEndomorphism_of_isNilpotent₂ R L M x y
+
+/-- This is barely worth having since it follows from `LieModule.traceForm_eq_zero_of_isNilpotent`
```

Comment:
Do you want this comment in the docstring? Or just in a regular comment?

Reply:
Good point, I don't want this to be a docstring!